lib/src/main/java/vmath/Mat2.java

```diff
@@ -9,4 +9,14 @@ public Mat2() {
             0.0f, 1.0f
         };
     }
+
+    public float get(int i, int j) {
+        if (i < 0 || i >= 2 || j < 0 || j >= 2) {
+            throw new IndexOutOfBoundsException("Invalid indices");
+        }
+
+        int index = i * 2 + j;
+
+        return m[index];
```

Comment:
Combine into one return statement like in other Matrix PRs.

Reply:
I corrected this as requested